### PR TITLE
Update deploy-interactive to include note on pods

### PR DESCRIPTION
**Pull request by Dean-Coakley**
Fixes: https://github.com/kubernetes/website/issues/18894

* Add note to what pods are during interactive deploy.

What do you think @sftim ?

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html [hunk 1]
```diff
@@ -17,6 +17,14 @@
 
     <main class="content katacoda-content">
 
+        <div class="row">
+            <div class="col-md-12">
+                <p>
+                    A Pod is the basic execution unit of a Kubernetes application–the smallest and simplest unit in the Kubernetes object model that you create or deploy. A Pod represents processes running on your Cluster. <a href="/docs/concepts/workloads/pods/pod-overview/#understanding-pods">Learn more about pods</a>
```
Comment (Dean-Coakley) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html]: This relative link didn't actually work in my testing.(Testing with a local Hugo environment) 

But I see similar relative urls elsewhere in the docs so I assumed an absolute url is not the way to go.

```suggestion
                    A Pod is the basic execution unit of a Kubernetes application–the smallest and simplest unit in the Kubernetes object model that you create or deploy. A Pod represents processes running on your Cluster. <a href="https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/#understanding-pods">Learn more about pods</a>
```

Comment (Dean-Coakley) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html]: Error with above relative url:
image

Comment (Dean-Coakley) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html]: Ah nice, it works in staging environment. 😄

Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html]: How about:
```suggestion
                    A Pod is the basic execution unit of a Kubernetes application. Each Pod represents a part of a workload that is running on your cluster. <a href="/docs/concepts/workloads/pods/pod-overview/#understanding-pods">Learn more about Pods</a>.
```
?

Comment (Dean-Coakley) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html]: Sounds good to me

**Pull request merged**